doc/classes/Bone2D.xml

```diff
@@ -15,7 +15,7 @@
 		<method name="apply_rest">
 			<return type="void" />
 			<description>
-				Stores the node's current transforms in [member rest].
+				Resets the bone to the rest pose. Exactly equivalent to [code]transform = rest[/code].
```

Comment:
We could be more explicit without needing a code sample.
```suggestion
				Resets the bone to the rest pose. This is equivalent to setting [member Node2D.transform] to [member rest].
```

Reply:
Seems fine, committed suggestion